pkg/command/types/v1alpha1/types.go

```diff
@@ -41,6 +41,7 @@ type OperatorCommandStatus struct {
 	CompletedAt *metav1.Time                `json:"completedAt,omitempty"` // CompletedAt is the time at which the command was completed
 	Executer    string                      `json:"executer,omitempty"`    // Executer is the entity that executed the command
 	Error       *OperatorCommandStatusError `json:"error,omitempty"`       // Error is the error that occurred during the execution of the command (if any)
+	Payload     []byte                      `json:"payload"`               // Payload is the response payload from execution of the command (if any)
```

Comment:
Please add omitempty

Reply:
You also need to update the CRD https://github.com/kubescape/helm-charts/tree/main/charts/dependency_chart/operatorcommand-crds

Reply:
https://github.com/kubescape/helm-charts/pull/536